angular/src/app/shared/paginator/paginator.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="paginator">
-  <button mat-button
+  <button *ngIf="showPrevious()" mat-button
```

Comment:
`this.paginationType !== PaginationTypes.UNIDIRECTION` seems fairly easy to be put directly here

Reply:
Taking this back, after investigation this won't work. must be property of component